src/sidebar/util/is-third-party-service.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
I couldn't find a better place to put this file, without first doing some reorganising that I don't feel I have time for.

The similar function `isThirdPartyUser()` is in `account-id.js` but this file is just for account stuff.

Other service-related stuff is in `service-config.js` and `service-url.js`. I think what's probably needed is an `AnnotationService` class that contains everything in `service-config.js`, `service-url.js`, `is-third-party-service.js` and probably lots of other stuff besides.

Reply:
>  I think what's probably needed is an AnnotationService class that contains everything

I don't think a class would be necessary, just a module of service-related utility functions would be fine. eg. In `util/service.js`.

Reply:
It'd be nice to only have to construct an object once, passing the settings for a particular service (from the services config) into the constructor, as well as any other dependencies it needs, and then be able to access things like `serviceUrl()`, `serviceConfig()` and `isThirdPartyUser()` on that object without having to pass in the service config settings and other dependencies every time. 

This is just nicer code, in my opinion, than the approach where each user module has to hold on to the service config settings and other dependencies and make sure to pass the right params to each service-related utility function each time it calls one.

If we end up adding support for multiple annotation services to the client (which the services array was anticipating) it'll probably be even nicer, as you can instantiate one `AnnotationService` object for each service. (Although I think we probably want to add multiple services support to h, not to the client.)